src/pages/hyunjaeKim/Main/MainArea/Contents/DetailWrapper.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Replies from "../Contents/Replies/Replies";
+
+export default class DetailWrapper extends Component {
+  state = {
+    likes: 1515,
+    isRed: false,
+  };
+
+  handleDelete = reply => {
+    this.props.onDelete(reply);
+  };
+
+  handleBtn = () => {
+    this.setState({ ...this.state, isRed: !this.state.isRed }, () => {
+      this.handleIncrement();
+    });
+  };
+
+  handleIncrement = () => {
+    this.state.isRed === true
+      ? this.setState({ ...this.state, likes: this.state.likes + 1 })
+      : this.setState({ ...this.state, likes: this.state.likes - 1 });
+  };
```

Comment:
```suggestion
handleIncrement = () => {
  const { isRed, likes } = this.state
  
  const sign = { true: 1, false: -1 }
  
  this.setState((prev) => ({
    ...prev,
    likes: likes + sign[isRed]
  }))  
};
```
- 함수 내 데이터가 외부 값을 바라보기 보다는 최대한 순수하게 함수 내부의 값을 사용 (ex. this.state -> prev)
- this.setState는 등은 갇은 동작이라면 처음 데이터가 들어가는 거겠죠.

Reply:
넵 함수 내 데이터는 최대한 함수 내부의 값을 사용하도록 주의하겠습니다! 동 건도 반영 완료입니다:)